api.R

```diff
@@ -11,91 +12,130 @@ source("generics.R")
 # Read AWS creds from Docker secrets
 if (file.exists("/run/secrets/ENV_FILE")) {
   readRenviron("/run/secrets/ENV_FILE")
-} else {
+} else if (file.exists("secrets/ENV_FILE")) {
   readRenviron("secrets/ENV_FILE")
+} else {
+  readRenviron(".env")
 }
-readRenviron(".env")
 
 # Get the model run attributes at runtime from env vars
 dvc_bucket <- Sys.getenv("AWS_S3_DVC_BUCKET")
 run_bucket <- Sys.getenv("AWS_S3_MODEL_BUCKET")
-run_id <- Sys.getenv("AWS_S3_MODEL_RUN_ID")
-run_year <- Sys.getenv("AWS_S3_MODEL_YEAR")
 api_port <- as.numeric(Sys.getenv("API_PORT", unset = "3636"))
+default_run_id_var_name <- "AWS_S3_MODEL_RUN_ID"
```

Comment:
It might be nice to change the name of this env var to something that more clearly marks it as the default (like `AWS_S3_DEFAULT_MODEL_RUN_ID`), but keeping the same name for now means one fewer thing we have to change during deployment. I'm open to changing it now if you feel strongly about it, however.

Reply:
Done in abe540391c05b750507e5086fedbf27b31c49d51, with an updated list of deploy steps in the PR body.